cpu/stm32/Makefile.dep

```diff
@@ -8,4 +8,8 @@ ifneq (,$(filter periph_usbdev,$(FEATURES_USED)))
   USEMODULE += xtimer
 endif
 
+ifneq (,$(filter usb_board_reset,$(USEMODULE)))
+  DEFAULT_MODULE += bootloader_stm32
```

Comment:
```suggestion
  USEMODULE += bootloader_stm32
```

Reply:
The thing is, I'd like to be able to disable this.
Not every STM32 part implements the USB DFU bootloader. The stm32f103 on the popular bluepill boards only has UART mode, so that doesn't help you if you only have USB.

Here a software bootloader would have to be used instead (and the board would then `DISABLE_MODULE += bootloader_stm32`.

Reply:
And I would not include this module all the time when `usb_board_reset` is used. Just when `usb_board_reset` and no bootloader other than `bootloader_stm32` is used. It could happen that an arduino bootloader is pre-flashed or something else (see the OpenCM9 board for instance).

```mk
ifneq (,$(filter usb_board_reset,$(USEMODULE)))
  ifeq (,$(filter-out bootloader_stm32,$(filter bootloader_%,$(USEMODULE))))
    USE_MODULE += bootloader_stm32
  endif
endif

(untested)
```

Reply:
> Here a software bootloader would have to be used instead 

Another bootloader will be used by the board so this will disable the use of `bootloader_stm32`. `DEFAULT_MODULE` seems error prone here.